core/miner/storage_fsm/impl/precommit_batcher_impl.cpp

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "miner/storage_fsm/impl/precommit_batcher_impl.hpp"
+
+#include <utility>
+
+namespace fc::mining {
+
+  PreCommitBatcherImpl::PreCommitBatcherImpl(size_t maxTime,
+                                             std::shared_ptr<FullNodeApi> api)
+      : api_(std::move(api)){};
+
+  PreCommitBatcherImpl::preCommitEntry::preCommitEntry(
+      primitives::TokenAmount number, api::SectorPreCommitInfo info)
+      : deposit(std::move(number)), pci(std::move(info)){};
+
+  PreCommitBatcherImpl::preCommitEntry &
+  PreCommitBatcherImpl::preCommitEntry::operator=(const preCommitEntry &x) =
```

Comment:
Please use "lhs", "rhs", "other" for such parameters, not "x"

Reply:
fixed